HacktoberfestProject.Web/Models/Entities/PrEntity.cs

```diff
@@ -3,13 +3,15 @@
 	public class PrEntity
 	{
 		public int PrId { get; set; }
+
+		public string Title { get; set; }
```

Comment:
What does the Title field represent?

Reply:
The title of the PR. Like the title of this PR is *First changes to enable us to store and retrieve data*. This might be more visually appealing to show than just the Url or Id